dali/ft/filecopy.cpp

```diff
@@ -2596,6 +2639,325 @@ void FileSprayer::pushParts()
     performTransfer();
 }
 
+using namespace Azure::Storage;
+using namespace Azure::Storage::Files;
+using namespace Azure::Storage::Blobs;
+
+enum ApiCopyStatus { Pending, Success, Failed, Aborted };
+interface IAPICopyClient : implements IInterface
```

Comment:
refactor: Not for this PR, but as part of the next stage can any azure api-related functionality be split out into a separate source file.  (Common api related functionality, like this interface, can stay here or in a common header.)

Reply:
Will do: https://track.hpccsystems.com/browse/HPCC-28147